### ISSUE TITLE
Simple Dll Still does not work

### ISSUE THREAD

**Issue by vankuih** [ddxoft/master#31]
I'm starting it as administrator and have secure boot off and it still doesn't work

**Comment by ddxoft** [comment]
win10 home version?


vankuih ***@***.***> 于2022年1月19日周三 03:16写道：

**Comment by vankuih** [comment]
yes

**Comment by Vitokonuk** [comment]
Yeah it's not working for me too. I using it in Python also i have windows 10 pro

**Comment by ddxoft** [comment]
What hint?
on windows remote desktop?

Vitokonuk ***@***.***> 于2022年1月21日周五 11:35写道：

**Comment by Vitokonuk** [comment]
No
